test_scripts/add_csv_data_source.sh

```diff
@@ -0,0 +1,3 @@
+source vars.txt
+echo  $SEARCHENGINE_IMAGE
+sudo docker run -v /data/searchengine/searchengine/:/etc/searchengine/  -v /data/searchengine/searchengine/logs/:/opt/app-root/src/logs/ --network searchengine-net $SEARCHENGINE_IMAGE set_data_source_files -n test_csv
```

Comment:
``test_csv`` should be replaced by $DATA_SOURCE cf. slack comment ``modify the script to replace test_csv with bia.`` so that the script does not need to be updated